README.adoc

```diff
@@ -60,47 +62,51 @@ If you wish to use different Infinispan version than provided by this starter, p
        </dependency>
     </dependencies>
  </dependencyManagement>
+----
 
-```
-
-## Using Embedded mode
-
-The starters operate in Client/Server mode by default. Enabling Embedded mode requires adding `infinispan-core`
-to the classpath.
-
-Once you add infinispan's `spring-boot-starter` as a dependency to your project, a bean of type `EmbeddedCacheManager` can
-be `autowired` into your java configuration classes. Just simply add:
+== Running in Embedded Mode
 
-```java
+. Add `infinispan-core` to your project's classpath.
++
+This starter operates in Remote Client/Server mode by default. Adding `infinispan-core` to the classpath enables Infinispan to run in Embedded mode.
```

Comment:
infinispan-spring-boot-starter-embedded or infinispan-spring-boot-starter-remote, instead of core.

Reply:
@karesti need to clarify this, please. if this is to enable embedded mode, you should add `infinispan-spring-boot-starter-embedded` here, right? it's `infinispan-spring-boot-starter-remote` by default?